supervisor/api/__init__.py

```diff
@@ -379,6 +379,10 @@ def _register_addons(self) -> None:
             [
                 web.get("/addons", api_addons.list),
                 web.get("/addons/{addon}/info", api_addons.info),
+                web.get("/addons/{addon}/icon", api_addons.icon),
+                web.get("/addons/{addon}/dark_icon", api_addons.dark_icon),
+                web.get("/addons/{addon}/logo", api_addons.logo),
+                web.get("/addons/{addon}/dark_logo", api_addons.dark_logo),
```

Comment:
move that into legacy part on store

Reply:
no?
/addons/* are used for installed addons
/store/addons/* are used for not installed

Reply:
see comment above

Reply:
@pvizeli I'm confused too, why would this be legacy? Installed addons can have a dark icon/logo too